testsuite/MDAnalysisTests/utils/test_streamio.py

```diff
@@ -226,8 +225,6 @@ def test_expandvars(self):
         assert_equal(value, reference,
                      err_msg="os.path.expandvars() did not expand HOME")
 
-    # Segmentation fault when run as a test on Mac OS X 10.6, Py 2.7.11 [orbeckst]
-    @dec.skipif(True)
```

Comment:
I would prefer an issue rather than just copying the comment as you did. Since travis broke on linux with the old numpy, the comment is obviously missing part of the problem.